src/components/TodoList.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {/* This is a completed todo */}
+
+      {/* <div data-cy="Todo" className="todo completed">
+        <label className="todo__status-label">
+          <input
+            data-cy="TodoStatus"
+            type="checkbox"
+            className="todo__status"
+            checked
+          />
+        </label>
+        <span data-cy="TodoTitle" className="todo__title">
+          Completed Todo
+        </span> */}
+
+      {/* Remove button appears only on hover */}
+
+      {/* <button type="button" className="todo__remove" data-cy="TodoDelete">
+          ×
+        </button> */}
+
+      {/* overlay will cover the todo while it is being deleted or updated */}
+
+      {/* <div data-cy="TodoLoader" className="modal overlay">
+          <div className="modal-background has-background-white-ter" />
+          <div className="loader" />
+        </div>
+      </div> */}
```

Comment:
Remember to get rid of comments before making a `Pull Request` next time.

Reply:
> Remember to get rid of comments before making a `Pull Request` next time.

Good point